src/System.Linq/src/System/Linq/Select.cs

```diff
@@ -170,9 +172,58 @@ public int GetCount(bool onlyIfCheap)
             {
                 return _source.Length;
             }
+
+            public IPartition<TResult> Skip(int count)
+            {
+                if (count == 0) return (IPartition<TResult>)Clone();
+                if (count >= _source.Length) return new EmptyPartition<TResult>();
```

Comment:
That's a question I've been meaning to ask you, since the increase in uses of `Array.Empty`. I'd be happier if we did, but leant on the side of caution in terms of object identity.